src/App.tsx

```diff
@@ -1,6 +1,13 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
+
+enum SortDirection {
+  alph = 'is-info',
+  len = 'is-success',
+  rev = 'is-warning',
+}
```

Comment:
enum's elements should be in a CamelCase 
```suggestion
enum SortDirection {
  Alph = 'is-info',
  Len = 'is-success',
  Rev = 'is-warning',
}
```